apps/api/e2e/tests/multi-agent-competition.test.ts

```diff
@@ -366,179 +314,4 @@ describe("Multi-Agent Competition", () => {
       }
     }
   });
-
-  // Test that portfolio values change over time due to price fluctuations
-  test("portfolio values should change differently for agents holding different tokens", async () => {
```

Comment:
this seemed like an unnecessary test since we do so many similar checking elsewhere